ZIGSIMPlus/Models/Stores/StoreManager.swift

```diff
@@ -54,9 +54,9 @@ class StoreManager {
         ))
 
         // Add data from stores
-        bundle.elements += LocationDataStore.shared.toOSC()
-        bundle.elements += TouchDataStore.shared.toOSC()
-        bundle.elements += MiscDataStore.shared.toOSC()
+//        bundle.elements += LocationDataStore.shared.toOSC()
+//        bundle.elements += TouchDataStore.shared.toOSC()
+//        bundle.elements += MiscDataStore.shared.toOSC()
```

Comment:
👀？ Does this cause errors?

Reply:
Oops, I mistakenly pushed the change.
I'm going to revert it.